src/components/StaticSuperscript.tsx

```diff
@@ -12,47 +15,53 @@ const StaticSuperscript = React.forwardRef<
     show?: boolean
     hideZero?: boolean
     absolute?: boolean
+    thoughtId?: ThoughtId
   }
->(({ n, style, show = true, hideZero, absolute, cssRaw }, forwardRef) => (
-  <span
-    ref={forwardRef}
-    className={css(
-      {
-        /* prevent expanded click area from wrapping */
-        whiteSpace: 'nowrap',
-        zIndex: 'stack',
-        pointerEvents: 'none',
-        userSelect: 'none',
-      },
-      cssRaw,
-    )}
-    style={style}
-  >
-    {show && (
-      <span
-        className={css({
-          fontSize: '60%',
+>(({ n, style, show = true, hideZero, absolute, cssRaw, thoughtId }, forwardRef) => {
+  const fill = useSelector(state => getElementFill(state, thoughtId!))
```

Comment:
`!` overrides the type system and should only be used in rare cases where the truthiness of a value is guaranteed for reasons that the type system cannot capture.

Instead, check `thoughtId` manually or change `getElementFill` to take undefined.